packages/sdk/src/oracle/services/statemachines/setActiveRequest.ts

```diff
@@ -0,0 +1,38 @@
+import { Update } from "../update";
+import Store from "../../store";
+import { Inputs } from "../../types/state";
+import { Handlers as GenericHandlers } from "../../types/statemachine";
+import { getFlags } from "../../utils";
+
+// required exports for state machine
+export type Params = Inputs["request"];
+export type Memory = { error?: Error };
+export function initMemory(): Memory {
+  return {};
+}
+export function Handlers(store: Store): GenericHandlers<Params, Memory> {
+  const update = new Update(store);
+  return {
+    async start(params: Params, memory: Memory) {
```

Comment:
by default all contexts start in the "start" state. so this is where you would put the initial logic